addons/packages/vsphere-pv-csi/2.4.0/bundle/config/overlays/set-cns-csi-conf.yaml

```diff
@@ -0,0 +1,8 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/values.star", "values")
+
+#@overlay/match by=overlay.subset({"kind": "ConfigMap", "metadata": {"name": "pvcsi-config"}}),expects=1
+---
+data:
+  #@overlay/replace
+  cns-csi.conf: #@ "[GC]" + "\nendpoint = %s\nport = %d\ntanzukubernetescluster-uid = %s\ntanzukubernetescluster-name = %s\ncluster-distribution = \"TKGService\""%(values.vspherePVCSI.supervisor_master_endpoint_hostname,values.vspherePVCSI.supervisor_master_port,values.vspherePVCSI.cluster_uid,values.vspherePVCSI.cluster_name)
```

Comment:
This line is a bit different from the template of 2.4.1, please confirm it's expected.

```
2.4.1:
cns-csi.conf: #@ "[GC]" + "\nendpoint = %s\nport = %d\ntanzukubernetescluster-uid = %s\ntanzukubernetescluster-name = %s"%(values.vspherePVCSI.supervisor
_master_endpoint_hostname,values.vspherePVCSI.supervisor_master_port,values.vspherePVCSI.cluster_uid,values.vspherePVCSI.cluster_name)
```

Reply:
Yes thats intentional. cluster-distribution = "TKGService" is needed.